docs/process.rst

```diff
@@ -219,8 +219,7 @@ Utopic) will run into problems with running the functional tests.
   string that combines the value in the branch's ``CHANGES.yaml`` with the
```

Comment:
This file is massively out of date; I corrected the bits relevant to this branch, but someone should take a look at this whole file.